Modules/binascii.c

```diff
@@ -367,7 +368,10 @@ binascii_b2a_uu_impl(PyObject *module, Py_buffer *data)
         return NULL;
 
     /* Store the length */
-    *ascii_data++ = ' ' + (bin_len & 077);
+    if (backtick)
```

Comment:
It's good. The code seems more clear telling the logic that only zero needs to be specially treated.